File: src/main/java/italo/scm/model/Endereco.java
package italo.scm.model;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@Getter
@Setter
@Entity
@Table(name="endereco")
public class Endereco {

	@Id
	@GeneratedValue(strategy=GenerationType.IDENTITY)
	private Long id;
	
	private String logradouro;
	
	private String numero;
	
	private String bairro;
	
	private String cidade;
	
	private String uf;
	
}

File: frontend/src/app/bean/endereco/endereco.ts
import { Municipio } from "./municipio";
import { Uf } from "./uf";

export interface Endereco {
    id : number;
    logradouro : string,
    numero : string;
    bairro : string;
    municipio : Municipio;
    uf : Uf;
}